706.py

```diff
@@ -0,0 +1,49 @@
+class MyHashMap:
+
+    def __init__(self):
+        """
+        Initialize your data structure here.
+        """
+        self.space_size = 9999
+        self.space = [[] for _ in range(self.space_size)]
+
+    def put(self, key: int, value: int) -> None:
+        """
+        value will always be non-negative.
+        """
+        hashcode = key % self.space_size
```

Comment:
This line of code is repeated three times in this class. We should extract it into its own function for maintainability (e.g., what if we later decided to use a different hash function for this hash map design?).